tests/src/test/java/com/ibm/watson/developer_cloud/discovery/v1/DiscoveryServiceIT.java

```diff
@@ -132,6 +132,8 @@ public static void setupClass() {
                     .build();
             CreateEnvironmentResponse createResponse = discovery.createEnvironment(createRequest).execute();
```

Comment:
this line was failing if you didn't already have an environment created before starting these tests